### ISSUE TITLE
The policies directory should be moved inside the Core package

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#2515]
I fail to see the benefits of having the `Policies` package inside the root directory of `Tribler`. We also have a cyclic dependency now between the `Core` and `Policies` package.

Since the `Policies` package is part of the credit mining mechanism (which is essentially part of libtribler), I think it makes more sense if we move this directory inside the `Core`.

@ardhipoetra any thoughts on this?

**Comment by ardhipoetra** [comment]
I don't mind. To be frank, the name `Policies` not really descriptive. I thought changing it into `CreditMining`, or something.

On the way to that, I think #2398 should be merged first to remove `Main` dependencies. It depends on another PR #2463, though.

**Comment by whirm** [comment]
I would move everything related to credit mining into a single package inside of `core`

**Comment by devos50** [comment]
@ardhipoetra if you have some time, can you take care of this?
